tagtester/main.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+import logging
+import os
+
+from tagcompare import placelocal
+from tagcompare import webdriver
+from tagcompare import output
+from tagcompare import settings
+from tagcompare import capture
+from tagcompare import image
+
+
+MODULE_NAME = 'tagtester'
+OUTPUT_BASEDIR = os.path.join(settings.HOME_DIR, MODULE_NAME)
+SETTINGS = {
```

Comment:
TODO - this should be stripped out to be its own json file, where each json file represent a test or a set of tests